app/components/edit/ProvidedService.jsx

```diff
@@ -14,7 +14,7 @@ const InputField = ({
     <input
       type={type}
       placeholder={placeholder}
-      value={value}
+      value={value || ''}
```

Comment:
Do you still require this when you've already set the empty string as the default argument in `defaultProps`?

Reply:
Yeah, apparently React chooses to [handle `null` differently from `undefined`](https://github.com/facebook/react/issues/2166), and only `undefined` or not attempting to pass in the prop at all will trigger the default prop. If you pass in a `null`, then it will still get the value of null.